README.md

```diff
@@ -6,7 +6,7 @@
 
 [![Latest published parley version.](https://img.shields.io/crates/v/parley.svg)](https://crates.io/crates/parley)
 [![Documentation build status.](https://img.shields.io/docsrs/parley.svg)](https://docs.rs/parley)
-[![Dependency staleness status.](https://deps.rs/repo/github/linebender/parley/status.svg)](https://deps.rs/repo/github/linebender/parley)
```

Comment:
Yeah I think not having a top level crate might be the way to go for repos that have more than one publishable crate. The reasoning for this gets stronger with every additional crate in the repo. The dependency badge is just one thing, but it would be nice to have all sorts of info and discussion about the repo and the crates in it - but none of that really fits the top level crate's readme on crates.io etc.

Let's discuss this further in the upcoming badges etc discussion. For now I'll merge this as-is and we'll change it later to whatever the big decision will be.